std/uni.d

```diff
@@ -7777,20 +7777,26 @@ if (isInputRange!S1 && isSomeChar!(ElementEncodingType!S1)
     && isInputRange!S2 && isSomeChar!(ElementEncodingType!S2))
 {
     import std.internal.unicode_tables : sTable = simpleCaseTable; // generated file
-    import std.utf : byDchar;
-
-    auto str1 = r1.byDchar;
-    auto str2 = r2.byDchar;
+    import std.utf : decodeFront;
+    import std.typecons : Yes;
+    static import std.ascii;
```

Comment:
Why not a selective and renamed import ?

Reply:
I like explicit std.ascii prefix whenever there is toLower/toUpper and friends